.github/workflows/cron_pre_release.yaml

```diff
@@ -0,0 +1,31 @@
+name: Publish Pre-Release
+on:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
what happens if the code didn't change, will this run and bail? 🤔

Reply:
Yes, there's a skip duplicates flag in the NuGet publish script. If nothing changes this will generate a version with the same number and nothing will happen

Reply:
Sounds good!